meson.build

```
@@ -1,7 +1,7 @@
 project(
   'rauc',
   'c',
-  version : '1.14',
+  version : '1.15',
```

In the previous iteration, we used something like:

```suggestion
  version : '1.15-dev',
```